### PR TITLE
feat: Add the webgen section in projects

### PR DESCRIPTION
**Pull request by Dharmesh-Poddar**
#207 
preview link: https://agitated-bartik-89f983.netlify.com/

Screenshot from 2019-09-08 15-04-38

### CODE REVIEW

**Comment by Dharmesh-Poddar** [comment]
@mariobehling @liveHarshit @uds5501 please review it.

File: index.html [hunk 1]
```diff
@@ -533,6 +533,10 @@ <h3>Voicerepublic</h3>
 						</div>
 					</div>
 
+
+
+
+
```
Comment (ShridharGoel) [inline index.html]: Revert this.

**Comment by ShridharGoel** [comment]
Also, use a better image instead of an image which contains just the name of the project.

**Comment by Dharmesh-Poddar** [comment]
@shivaylamba should I use this image .

**Comment by Dharmesh-Poddar** [comment]
@shivaylamba will it be okay if I use this image.

**Comment by liveHarshit** [comment]
> @shivaylamba will it be okay if I use this image.

Yes, you should use it, but take care of the card size as others.

**Comment by shivaylamba** [comment]
@dharmesh-poddar Keep the width and height as 679 and 485 respectively

**Comment by Dharmesh-Poddar** [comment]
@shivaylamba should I use the inline style for this image as the height and width that  you have specified are not compatible with each other on any image resizer available.

File: index.html [hunk 2]
```diff
@@ -650,6 +654,20 @@ <h3>Badgeyay</h3>
...
+         	<div class="col-md-4 col-sm-6">
+						<div class="info-box">
+							<img alt="WebsiteGenerator" src="img/websitegen.png">
+							<h3>Website Generator</h3>
+							<p>The web generator application can generate event websites by getting data from event JSON files and binary media files, that are stored in a compressed zip file. You can also access the application through a REST API. Websites that are generated by the "web app generator" can be uploaded to any web location, e.g. on Github pages or any server (e.g. via ftp).</p></p><div><br><br><br></div>
+							<div class="text-link">
+								<a href="http://opev-webgen-dev.herokuapp.com/" target="_self">Website</a>
+								<a href="https://gitter.im/fossasia/open-event-webapp" target="_self">Chat</a>
+								<a href="https://groups.google.com/forum/#!forum/open-event" target="_self">Mailing List</a>
+								<a href="https://github.com/fossasia/open-event-wsgen" target="_self">Repositories</a>
```
Comment (uds5501) [inline index.html]: There is only one repo so `repository` would be fine.

File: index.html [hunk 3]
```diff
@@ -650,6 +654,20 @@ <h3>Badgeyay</h3>
 					</div>
 
 
+         	<div class="col-md-4 col-sm-6">
+						<div class="info-box">
+							<img alt="WebsiteGenerator" src="img/websitegen.png">
```
Comment (uds5501) [inline index.html]: Update the image

**Comment by Dharmesh-Poddar** [comment]
@shivaylamba the dimension that you have specified are different from other projects images ,are you sure I should go with it.

File: index.html (hunk 3 above)
Comment (Dharmesh-Poddar) [reply index.html]: @uds5501  Thanks for the review . I will make the changes.

**Comment by shivaylamba** [comment]
Yes, Actually I did realize that a bit later. What I would suggest is look for the size of any one of the existing pictures and scale yours accordingly. For ex. that of Eventyay image.

**Comment by Dharmesh-Poddar** [comment]
@shivaylamba please check the new preview link : https://reverent-jones-0d3e3d.netlify.com

**Comment by Dharmesh-Poddar** [comment]
@liveHarshit can you provide me the image that I need to use

**Comment by liveHarshit** [comment]
> @liveHarshit can you provide me the image that I need to use

You can use the same, or make some edits. For maintaining the ratio, crop it instead of stretching the original pixels.

**Comment by Dharmesh-Poddar** [comment]
@liveHarshit I am a bit confused about the image as you said it is okay if the image doesnot come in the frame and in previous comments everyone said to keep the image as other project images.

**Comment by mariobehling** [comment]
@Dharmesh-Poddar Images should never be stretched or squeezed. If you understand any comment like that you misunderstand.

File: index.html [hunk 4]
```diff
@@ -649,8 +649,20 @@ <h3>Badgeyay</h3>
 						</div>
 					</div>
 
-
-
+                    <div class="col-md-4 col-sm-6">
+						<div class="info-box">
+							<img alt="WebsiteGenerator" src="img/webgen.png">
+							<h3>Website Generator</h3>
```
Comment (kushthedude) [inline index.html]: Make it more clear, Website Generator is ambiguous.

**Comment by liveHarshit** [comment]
@Dharmesh-Poddar Any updates?

**Comment by Dharmesh-Poddar** [comment]
@liveHarshit I am stuck with the image part :(

**Comment by abishekvashok** [comment]
@Dharmesh-Poddar try to get some images from the repo or fossasia.org

**Comment by mariobehling** [comment]
Closing due to inactivity.

**Pull request closed without merging**